apps/frontend/src/libs/components/page-layout/page-layout.tsx

```diff
@@ -0,0 +1,27 @@
+import { SIDEBAR_ITEMS } from "~/libs/constants/constants.js";
+
+import { Header } from "../header/header.js";
+import { Sidebar } from "../sidebar/sidebar.js";
```

Comment:
I think we might need to prefer absolute imports here. 
```suggestion
import { Header, Sidebar } from "~/libs/components/components.js";
```

Reply:
Yes it is, but I don't think it is a big deal here. Usually we use relative imports for cases if the file is in the same, or sub directory (so, strictly dependent). Common components should be standalone (kind of) and if they depend on other common ones we import them using absolute imports. (It's reliable, improves component maintainability, and is useful for example if we move component somewhere).

Reply:
yes, let's use absolute import here

Reply:
changed